.github/workflows/add-release-info-to-pyxis.yml

```diff
@@ -0,0 +1,42 @@
+name: Add Release Info to Pyxis
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+      commit:
+        required: true
+        type: string
+      host:
+        required: true
+        type: string
+    secrets:
+      certificate:
+        required: true
+      key:
+        required: true
+      certificatePassword:
+        required: true
+
+jobs:
+  call-pyxis-endpoint:
+    name: Add Release Info
+    runs-on: ubuntu-latest
+    steps:
+      - name: Save Certificate and Key
+        run: |
+          echo -en "${{ secrets.certificate }}" > /tmp/preflight-release.crt
+          echo -en "${{ secrets.key }}" > /tmp/preflight-release.key
```

Comment:
These echos will not expose the keys right? Also is there a need to delete these even though they are in tmp is there a legitimate concern they could be exposed if not manually removed?

Reply:
@acornett21 it does not hurt to remove it.

Create a final step for this job that `rm -f <files> || true` and make sure its if expression is set to `always()` so it runs regardless of any failure in this workflow.

Reply:
@komish I added a cleanup, let me know if this is correct.

### Info In GHA
```
Run rm -f /tmp/preflight-release.crt /tmp/preflight-release.key || true
  rm -f /tmp/preflight-release.crt /tmp/preflight-release.key || true
  shell: /usr/bin/bash -e {0}
  env:
    CERT_FILE_LOCATION: /tmp/preflight-release.crt
    KEY_FILE_LOCATION: /tmp/preflight-release.key
```